docs/how-to/tuning-guides.rst

```diff
@@ -0,0 +1,105 @@
+.. meta::
+   :description: AMD hardware optimization for specific workloads
+   :keywords: high-performance computing, HPC, Instinct accelerators, Radeon,
```

Comment:
perhaps remove tuning here and the other topics

Reply:
done. thanks.